third_party/terraform/resources/resource_dataflow_flex_template_job.go.erb

```diff
@@ -41,6 +41,13 @@ func resourceDataflowFlexTemplateJob() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"region": {
+				Type:        shema.TypeString,
+				Optional:    true,
```

Comment:
I'd recommend also marking the field as Computed so that if the user doesn't set region explicitly and we fall back to one configured provider-wide, Terraform doesn't think it needs to correct it to empty. We typically do this for region fields in just about every resource.